pkg/fleetautoscalers/fleetautoscalers_test.go

```diff
@@ -1950,7 +1951,7 @@ func TestApplyListPolicy(t *testing.T) {
 							}}}},
 			},
 			want: expected{
-				replicas: 0,
+				replicas: 1,
```

Comment:
@markmandel in the case where the autoscaler calculates percent buffer as 0 (when there are no allocated Counters / Lists) we manually change the desired buffer to 1. Should this be "limited", or not limited, or does it not matter?